shell/platform/glfw/flutter_glfw.cc

```diff
@@ -149,10 +149,59 @@ struct FlutterDesktopPluginRegistrar {
 
 // State associated with the messenger used to communicate with the engine.
 struct FlutterDesktopMessenger {
+  FlutterDesktopMessenger() = default;
+
+  void AddRef() { ref_count_.fetch_add(1); }
```

Comment:
AddRef and Release need declaration comments.

Reply:
done